dvc/commands/experiments/push.py

```diff
@@ -35,14 +35,17 @@ def run(self):
             jobs=self.args.jobs,
             run_cache=self.args.run_cache,
         )
-
-        if pushed_exps:
+        if pushed_exps := result.get("pushed"):
             ui.write(
                 f"Pushed experiment {humanize.join(map(repr, pushed_exps))} "
                 f"to Git remote {self.args.git_remote!r}."
             )
         else:
             ui.write("No experiments to push.")
+
+        if project_url := result.get("url"):
+            ui.write("[yellow]View your experiments at", project_url, styled=True)
```

Comment:
Can/Should we give a more specific URL?

Lile `showOnlyLive=1` parameter? 
Or even using the `pushed_exps` to include selection parameters in the URL

Reply:
When we push, it takes some time to parse. So the url may not be valid.

Reply:
@daavoo Do you mean a URL to specific experiments or something else? What do you think would be useful to show? Do you consider it a blocker?

Reply:
> @daavoo Do you mean a URL to specific experiments or something else? What do you think would be useful to show? 

I don't know, just something that takes me to see what I pushed. 
It feels kind of like if I use the GH CLI for `gh pr create` and I get the URL of the repo instead of the link to the P.R.

> Do you consider it a blocker?

Definitely not a blocker, just wanted to discuss